readme.md

```diff
@@ -106,8 +110,21 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.get('/', function (req, res) {
+  res.send('Hello World!');
+});
+
+app.post('/', function (req, res){
+  res.send('post request')
```

Comment:
Your syntax is all correct! But RESTful actions are things like Create, Edit, etc.